docs/governance.rst

```diff
@@ -0,0 +1,253 @@
+following system the following goals in mind:
+
+- Promote open discussions.
+- Minimize the administrative burden.
+- Strive for consensus.
+- Provide a path for when consensus cannot be achieved.
+- Grow the community.
+- Maximize the `bus factor`_ of the project.
+- Acknowledge the leadership and time multiple people contribute to our
+  community without demanding more time than any individual can offer.
+  Dividing leadership responsabilies into multiple smaller roles also
+  makes it easier to encourage new people to take on a leadership role
+  without fearing that too much work will be required of them.
+
+This governance structure is a work-in-progress. We welcome both people
+who want to take on a leadership role as well as ideas for our to improve
+this structure.
+
+
+
+Steering Committee and Leadership Roles
+---------------------------------------
+
+Organizational Principles
+`````````````````````````
+The steering committee steers. The goal of the steering committee is to help
+guide the direction of the project. Decisions in the steering committee will 
+focus on how to present project issues to the broader community in a clear way
+rather than making project decisions without community input. 
+
+Issues or pull requests that can be reviewed by any community member can be
+reviewed and approved by the steering committee or steering committee members.
+The steering committee can decide that an issue should be prioritized for wider
+communal discussion or that a pull request requires more discussion or reviews
+than standard before merging. The steering committee can also decide how a
+breaking change (something that changes existing user function calls or program
+outputs) will be presented to the developer and user base for discussion, before
+decisions are made.
+
+Steering committee decisions should strive for consensus and accept the
+majority’s decision. If voting is necessary, it can happen asynchronously and is
+defined as more than half of the members of the steering committee. If a member
+of the steering committee is unable to spend the time necessary to understand
+an issue & vote, they can recuse from a decision so that the number of committee
+members is reduced
+
+Openness is a priority:
+
+- Openness is critical to building trust with the broader community
+- Openness provides a mechanism for non-steering committee members to identify
+  and address steering committee blindspots
+- Openness provides a smoother transition to onboard future steering committee
+  members
+- If steering committee discussions (written or verbal) could include
+  community members without compromising efficiency, they should be invited. The
+  steering committee can schedule discussions without needing to ask about the
+  availability for people outside of the steering committee. If notes from
+  meetings can be openly shared without compromising personal privacy, they
+  should be.
```

Comment:
I might invert this to highlight that whenever possible, steering committee meetings will be open and minutes will be freely available. There are, of course, some circumstances when this will not be possible, but I think it's a reasonable default !